CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
 # Change Log
+## 1.8.0 - 2019-01-07
+### Added
+* Autodetect/install/update iotedgehubdev
+* Allow users to specify whether to install Node.js template
```

Comment:
by default node template is installed. And turn off install configuration currently is not widely used by user. It is a little bit advanced. So how about change to "Install Azure IoT Edge Node.js Module Generator automatically before new Node.js module?"

Reply:
Updated